doc/developer/requirements.md

```diff
@@ -127,3 +122,33 @@ You can also run any cozy-stack command with the `yarn cozy-stack` script, e.g.:
 ```
 yarn cozy-stack apps install --domain cozy.tools:8080 drive 'git://github.com/cozy/cozy-drive.git#build-drive'
 ```
+
+## cozy-desktop
+
+### Create a transifex API token
+
+- Create an account on http://www.transifex.com
+- Join the Cozy team
+- Get your API token from the account settings page
```

Comment:
It should not really be a requirement to build the app, assuming the translations in git are up-to-date.
And I wonder whether making it a requirement is packager-friendly...

Maybe we could just explain that the build script will try to update the translations when the `transifex` client is on the `PATH`, assuming one already has an account & the client is already set up with a token?
We could later add a doc explaining the transifex setup & workflow and add a link to it from here? WDYT?

Reply:
(we could also try to fix the npm script)

Reply:
Or just change the section title to:

```markdown
## Transifex (optional)
```

Reply:
I added this part because the setup procedure was leading to the question being raised and I had no idea what this was about.
Since this is optional, we should make sure the procedure doesn't lead to it unless we request it.